src/Symfony/Component/Validator/composer.json

```diff
@@ -19,7 +19,8 @@
         "php": "^7.1.3",
         "symfony/contracts": "^1.0.2",
         "symfony/polyfill-ctype": "~1.8",
-        "symfony/polyfill-mbstring": "~1.0"
+        "symfony/polyfill-mbstring": "~1.0",
+        "symfony/translation": "~4.2"
```

Comment:
I moved this into the `require` section since `LegacyTranslatorProxy` includes a reference still to the old interface

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Util/LegacyTranslatorProxy.php#L14